geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -10495,6 +10495,10 @@ public void addCacheServiceProfile(CacheServiceProfile profile) {
     this.cacheServiceProfiles.put(profile.getId(), profile);
   }
 
+  public void removeCacheServiceProfile(String profileId) {
```

Comment:
This API was already added as a part of GEODE-5109.
[Build will fail after merge with develop]